libs/vgc/ui/colorpalette.cpp

```diff
@@ -1744,8 +1744,6 @@ void ColorPaletteSelector::drawHueSelector_(core::FloatArray& a) {
     //   half-disk      rectangle        half-disk
     //
 
-    const core::Color borderColor(255, 255, 255); // temp
```

Comment:
You were asking why is Clang only complaining now about the unused borderColor: I think it's probably because you just made `Color(r, g, b)` constexpr, even though I don't understand why that should affect the static analysis of which variables are unused in this case. Clang might have thought there could be side effects, even though the definition of the function was inline.